packages/easy-coding-standard/src/Console/Command/MarkdownCodeFormatterCommand.php

```diff
@@ -4,146 +4,81 @@
 use Symplify\PackageBuilder\Console\ShellCode;
 use Symplify\SmartFileSystem\SmartFileInfo;
 use Symplify\SmartFileSystem\SmartFileSystem;
 
 final class MarkdownCodeFormatterCommand extends Command
 {
     /**
-     * @var SmartFileSystem
+     * @var string
      */
-    private $smartFileSystem;
+    private const SOURCE = 'markdown-file';
 
     /**
-     * @var FixerFileProcessor
-     */
-    private $fixerFileProcessor;
-
-    /**
-     * @var SniffFileProcessor
+     * @var SmartFileSystem
      */
-    private $sniffFileProcessor;
+    private $smartFileSystem;
 
     /**
-     * @var Configuration
+     * @var EasyCodingStandardStyle
      */
-    private $configuration;
+    private $easyCodingStandardStyle;
 
     /**
-     * @var EasyCodingStandardStyle
+     * @var MarkdownPHPCodeFormatter
      */
-    private $easyCodingStandardStyle;
+    private $markdownPHPCodeFormatter;
 
     public function __construct(
         SmartFileSystem $smartFileSystem,
-        FixerFileProcessor $fixerFileProcessor,
-        SniffFileProcessor $sniffFileProcessor,
-        Configuration $configuration,
-        EasyCodingStandardStyle $easyCodingStandardStyle
+        EasyCodingStandardStyle $easyCodingStandardStyle,
+        MarkdownPHPCodeFormatter $markdownPHPCodeFormatter
     ) {
         $this->smartFileSystem = $smartFileSystem;
-        $this->fixerFileProcessor = $fixerFileProcessor;
-        $this->sniffFileProcessor = $sniffFileProcessor;
-        $this->configuration = $configuration;
         $this->easyCodingStandardStyle = $easyCodingStandardStyle;
 
         parent::__construct();
+
+        $this->markdownPHPCodeFormatter = $markdownPHPCodeFormatter;
     }
 
     protected function configure(): void
     {
-        $this->setName('markdown-code-format');
-        $this->setDescription('Format markdown code');
-        $this->addArgument('markdown-file', InputArgument::REQUIRED, 'The markdown file');
+        $this->setName(CommandNaming::classToName(self::class));
+        $this->setDescription('Format Markdown PHP code');
+        $this->addArgument(self::SOURCE, InputArgument::REQUIRED, 'Path to the Markdown file');
     }
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         /** @var string $markdownFile */
-        $markdownFile = $input->getArgument('markdown-file');
-        if (! file_exists($markdownFile)) {
-            throw new NoMarkdownFileException(sprintf('Markdown file %s not found', $markdownFile));
-        }
-
-        $this->configuration->resolveFromArray(['isFixer' => true]);
-
-        $r = new ReflectionProperty($this->fixerFileProcessor, 'configuration');
-        $r->setAccessible(true);
-        $r->setValue($this->fixerFileProcessor, $this->configuration);
-
-        $r = new ReflectionProperty($this->sniffFileProcessor, 'configuration');
-        $r->setAccessible(true);
-        $r->setValue($this->sniffFileProcessor, $this->configuration);
```

Comment:
@samsonasik I've check this part... It was not needed, as `Configuration` is a service and its state is always the same. 

So this:

```php
$this->configuration->resolveFromArray(['isFixer' => true]);
```

set the value to `true` everywhere